docs/csharp/programming-guide/concepts/expression-trees/index.md

```diff
@@ -104,7 +104,7 @@ Console.WriteLine(factorial);
 // Prints 120.  
 ```  
   
- For more information, see [Generating Dynamic Methods with Expression Trees in Visual Studio 2010](http://go.microsoft.com/fwlink/?LinkId=169513), which also applies to later versions of Visual Studio.
+ For more information, see [Expression Trees (C#)](../../../../csharp/programming-guide/concepts/expression-trees/index.md), which also applies to later versions of Visual Studio.
```

Comment:
I just noticed we're linking to the topic itself. Sorry. 
So maybe let's just keep the change as you previously made here and in the vb topic and then we should check if we cover the subject of the blog post on the docs or not. I prefer not to link to stale content.